include/multigrid_pclomp/multigrid_ndt_omp.h

```diff
@@ -64,7 +64,7 @@
 #include <unsupported/Eigen/NonLinearOptimization>
 
 namespace pclomp {
-struct NdtResult {
+struct MGNdtResult {
```

Comment:
The `NdtResult` structure is used in `ndt_scan_matcher` in autoware.universe, so if you rename it, you need to change that at the same time.
https://github.com/autowarefoundation/autoware.universe/blob/a6003cf14458b0d9de8cc406f82a93b52da6fbee/localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp#L378

It would be a big change, so I would like to not change it if I can get away with it.

Is this name change really necessary?
I think it can be made common by using of the structure with the same name in ndt_omp.h.
I'll try this one if I have time.

Reply:
`NdtParams` has the same issue.

The name change is only necessary for including both `<pclomp/ndt_omp.h>` and `<multigrid_pclomp/multigrid_ndt_omp.h>`.

One possible solution is to put `struct NdtResult` inside class definition, and use it as 

```
using NormalDistributionsTransform =
    pclomp::MultiGridNormalDistributionsTransform<PointSource, PointTarget>;
using NdtResult=NormalDistributionsTransform::NdtResult
```

Reply:
Thank you for the modification. :+1: 
However, changing the structure name still affects autoware.universe, so it cannot be merged as is.
Please wait a moment as the team will decide whether to use the same NDT result structure in both classes or to separate the namespaces appropriately. :bow:

Reply:
We changed the NDTResult struct to be commonly used. I think it will work fine if you pull the latest tier4/main and resolve the conflict.
https://github.com/tier4/ndt_omp/pull/37

Reply:
:ok_hand: done